### PR TITLE
Fixing and adding features

### PR DESCRIPTION
**Pull request by jobo322**
1. fixed the conversion between with between inflection points and FWHM
2. It is possible to have different fwhm for gaussian and lorentzian contribution in pseudo voigt shape
3. there is a interval option to calculate the shape values

### CODE REVIEW

File: src/inflectionPointsWidthToFWHM.js [hunk 1]
```diff
@@ -18,9 +18,11 @@ export function inflectionPointsWidthToFWHM(
   if (typeof kind === 'string') kind = getKind(kind);
   switch (kind) {
     case 1:
```
Comment (lpatiny) [inline src/inflectionPointsWidthToFWHM.js:20]: Please use the constants GAUSSIAN, etc rather than 1,2,3

File: src/inflectionPointsWidthToFWHM.js [hunk 2]
```diff
@@ -18,9 +18,11 @@ export function inflectionPointsWidthToFWHM(
...
-      return Math.sqrt(2 * Math.LN2) * width;
+      //https://mathworld.wolfram.com/GaussianFunction.html
+      return width * Math.sqrt(2 * Math.LN2);
     case 2:
-      return width;
+      //https://mathworld.wolfram.com/LorentzianFunction.html
+      return width * Math.sqrt(3);
```
Comment (lpatiny) [inline src/inflectionPointsWidthToFWHM.js:25]: You could define a constant for Math.sqrt(3)

File: src/inflectionPointsWidthToFWHM.js [hunk 3]
```diff
@@ -18,9 +18,11 @@ export function inflectionPointsWidthToFWHM(
   if (typeof kind === 'string') kind = getKind(kind);
   switch (kind) {
     case 1:
-      return Math.sqrt(2 * Math.LN2) * width;
+      //https://mathworld.wolfram.com/GaussianFunction.html
+      return width * Math.sqrt(2 * Math.LN2);
```
Comment (lpatiny) [inline src/inflectionPointsWidthToFWHM.js:22]: Could define a constant for Math.sqrt(2 * Math.LN2)

File: src/inflectionPointsWidthToFWHM.js [hunk 4]
```diff
@@ -18,9 +18,11 @@ export function inflectionPointsWidthToFWHM(
...
     case 2:
-      return width;
+      //https://mathworld.wolfram.com/LorentzianFunction.html
+      return width * Math.sqrt(3);
     case 3:
       return width * (mu * (Math.sqrt(2 * Math.LN2) - 1) + 1);
```
Comment (lpatiny) [inline src/inflectionPointsWidthToFWHM.js:27]: Define a constant for Math.sqrt(2 * Math.LN2)

**Review by lpatiny — CHANGES_REQUESTED** [review]
Still don't see why we should add interval

**Comment by jobo322** [comment]
before we have a step size of one between shape values. It is needed to increment density density of values.

File: package.json [hunk 5]
```diff
@@ -9,6 +9,7 @@
     "src"
   ],
   "scripts": {
+    "testA": "jest -i src/__tests__/gaussian.test.js",
```
Comment (maasencioh) [inline package.json:12]: you could add [`.only`](https://jestjs.io/docs/en/api#testonlyname-fn-timeout) to a test to do this

**Pull request merged**